src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,43 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * This Deadline class implements the characteristics of a task
+ * that needs to be done before a specific date/time.
+ */
+public class Deadline extends Task {
+
+    protected LocalDate by;
```

Comment:
I thought this could be a little ambiguous. Perhaps a name such as 'timeBy' might be better.

Reply:
Good catch, I will change this variable name as per your suggestion! Thanks! 👍🏽